### PR TITLE
GKE platform setup: multicluster firewall rule

### PR DESCRIPTION
**Pull request by stevenctl**
cc @askmeegs @Shabirmean

### CODE REVIEW

File: content/en/docs/setup/platform-setup/gke/index.md [hunk 1]
```diff
@@ -79,3 +81,32 @@ Follow these instructions to prepare a GKE cluster for Istio.
         --clusterrole=cluster-admin \
         --user=$(gcloud config get-value core/account)
     {{< /text >}}
+
+## Multi-cluster communication
+
+In some cases, a firewall rule must be explicitly created to allow cross-cluster traffic
+
+{{< warning >}}
+The following instructions will allow communication for *all* clusters in your project. Adjust the commands as needed.
```
Comment (Shabirmean) [inline content/en/docs/setup/platform-setup/gke/index.md:90]:
```suggestion
The following instructions will allow communication between *all* clusters in your project. Adjust the commands as needed.
```

**Review by Shabirmean — APPROVED** [review]
LGTM

File: content/en/docs/setup/platform-setup/gke/index.md [hunk 2]
```diff
@@ -79,3 +79,32 @@ Follow these instructions to prepare a GKE cluster for Istio.
         --clusterrole=cluster-admin \
         --user=$(gcloud config get-value core/account)
     {{< /text >}}
+
+## Multi-cluster communication
+
+In some cases, a firewall rule must be explicitly created to allow cross-cluster traffic
```
Comment (ericvn) [inline content/en/docs/setup/platform-setup/gke/index.md:85]: To mimic the format of the rest of the document, you need a period after this sentence as well as the two numbered sentences.

Comment (stevenctl) [reply content/en/docs/setup/platform-setup/gke/index.md]: Done, thanks!

**Comment by ericvn** [comment]
Based on the issue talking about 1.9, I will mark this as going into 1.10 as well so it appears on istio.io.

**Review by ericvn — APPROVED** [review]

**Pull request merged**